### ISSUE TITLE
Unable to download wasm_fuzzer from google storage

### ISSUE THREAD

**Issue by rebeccasf** [nwjs/nw.js#7890]
NWJS Version : nw64
Operating System : Ubuntu 20.04

### Expected behavior

gclient sync --with_branch_heads should be able to pull the source code based on the branch checkout in nw.js, v8 and node.

### Actual behavior

~/nwjs  $ gclient sync --with_branch_heads
Updating depot_tools...
Syncing projects: 100% (130/130), done.
Running hooks:  32% (37/115) wasm_fuzzer
________ running 'python3 src/third_party/depot_tools/download_from_google_storage.py --no_resume --no_auth -u --bucket v8-wasm-fuzzer -s src/v8/test/fuzzer/wasm_corpus.tar.gz.sha1' in '/home/rebecca.chang/nwjs'
Fatal error: src/v8/test/fuzzer/wasm_corpus.tar.gz.sha1 not found when attempting enumerate files to download.
Error: Command 'python3 src/third_party/depot_tools/download_from_google_storage.py --no_resume --no_auth -u --bucket v8-wasm-fuzzer -s src/v8/test/fuzzer/wasm_corpus.tar.gz.sha1' returned non-zero exit status 1 in /home/rebecca.chang/nwjs
Fatal error: src/v8/test/fuzzer/wasm_corpus.tar.gz.sha1 not found when attempting enumerate files to download.

### How to reproduce

Follow the steps provided in this guide: https://nwjs.readthedocs.io/en/latest/For%20Developers/Building%20NW.js/

Process will stop at gclient sync step where wasm_fuzzer is not able to download.
Running hooks:  32% (37/115) wasm_fuzzer
________ running 'python3 src/third_party/depot_tools/download_from_google_storage.py --no_resume --no_auth -u --bucket v8-wasm-fuzzer -s src/v8/test/fuzzer/wasm_corpus.tar.gz.sha1' in '/home/rebecca.chang/nwjs'
Fatal error: src/v8/test/fuzzer/wasm_corpus.tar.gz.sha1 not found when attempting enumerate files to download.
Error: Command 'python3 src/third_party/depot_tools/download_from_google_storage.py --no_resume --no_auth -u --bucket v8-wasm-fuzzer -s src/v8/test/fuzzer/wasm_corpus.tar.gz.sha1' returned non-zero exit status 1 in /home/rebecca.chang/nwjs
Fatal error: src/v8/test/fuzzer/wasm_corpus.tar.gz.sha1 not found when attempting enumerate files to download.

->

**Comment by YLivay** [comment]
I'm seeing the same issue when building Windows on nw62.

**Comment by YLivay** [comment]
I don't know if this is will work because im still in the process of trying to build nwjs, but I got past this point with: `gclient sync --with_branch_heads --nohooks`, then clone the additional repos mentioned in the nwjs build instructions and then run `gclient runhooks`

**Comment by YLivay** [comment]
Make sure to clone it on the branch you want to build for. i just cloned with `--branch nw62` on all repos.

Also, I invote gn gen with the following arguments: `gn gen out/nw --args="is_debug = false is_component_ffmpeg = true target_cpu = ""x64"" symbol_level = 1 blink_symbol_level = 0"`. Still,  i'm not sure that actually worked since i'm still stuck building it myself haha.

Apparently vs2022 is not supported for gyp which is needed to build nw-node, so i've been going through the steps from the top.

**Comment by rebeccasf** [comment]
> Make sure to clone it on the branch you want to build for. i just cloned with `--branch nw62` on all repos.
> 
> Also, I invoke gn gen with the following arguments: `gn gen out/nw --args="is_debug = false is_component_ffmpeg = true target_cpu = ""x64"" symbol_level = 1 blink_symbol_level = 0"`. Still, i'm not sure that actually worked since i'm still stuck building it myself haha.
> 
> Apparently vs2022 is not supported for gyp which is needed to build nw-node, so i've been going through the steps from the top.

Yes, I am using all nw64 branches, but for the chromium src specifically, I need to checkout to a commit, so I use the nw64 HEAD and created a detach HEAD. This works for me. Proceed further, I am surprised to see it needs python2 to create the ninja files for node. With some changes on my host, managed to get it run with python2 but yes, more issues popping up.

Simply run the command in the doc, and gyp files wasn't generated. I will spend more time looking into this.
`~/nwjs-new/src ((HEAD detached at 9c94ffe8db43)) $ GYP_CHROMIUM_NO_ACTION=0 ./build/gyp_chromium -I third_party/node-nw/common.gypi -D building_nw=1 -D clang=1 third_party/node-nw/node.gyp
Updating projects from gyp files...
gyp: name 'target_arch' is not defined while evaluating condition 'target_arch=="x64" or target_arch=="arm64"' in third_party/node-nw/node.gyp
`

**Comment by YLivay** [comment]
The docs are outdated, tho I got farther by looking at the commands as they are shown in their buildbot CI page, but even then, and with correct environment set up and all I couldn't get it working.

Got stuck on the step to compile node-nw. its complaining about not being able to find icudt.dll, which i understand should have been vendored in and not built.. the flags to disable building it and instead using icudt.dat files seem to uhh... not actually work, so im not sure whats wrong.

In the end, it takes too long to compile and try things blindly on my potato of a PC so I gave up.. I wish I had some more useful info to share or a PR to contribute to updating the docs but I don't haha.
